src/components/ToDoForm.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+export default function ToDoForm({ addToDo }) {
+  const [toDo, setToDo] = useState('');
+
+  const setStateOnSubmit = (e) => {
+    e.preventDefault();
+    addToDo(toDo);
+    setToDo('');
+  };
+  return (
+    <form onSubmit={setStateOnSubmit} className="todo-form">
+      <input
+        className="todo-input"
+        onChange={(e) => setToDo(e.target.value)}
+        type="text"
+        value={toDo}
+      />
+      <button className="todo-button" type="submit">
```

Comment:
there's a bug in this app in that it allows you to submit a blank todo. Think of a way to not allow this